purchases/src/test/java/com/revenuecat/purchases/utils/Responses.kt

```diff
@@ -106,7 +136,12 @@ object Responses {
                         "period_type": "normal",
                         "ownership_type": "PURCHASED",
                         "store_transaction_id" : "GPA.3394-7009-4518-41945..8",
-                        "expires_date": "${Iso8601Utils.format(threeMonthFreeTrialExpirationDate)}"
+                        "expires_date": "${Iso8601Utils.format(threeMonthFreeTrialExpirationDate)}",
+                        "display_name": "Basic Monthly",
+                        "price": {
+                          "currency": "USD",
+                          "amount": "5.99"
+                        }
```

Comment:
Should we have some sample response with `auto_resume_date` and `product_plan_identifier`?